libminifi/include/utils/StringUtils.h

```diff
@@ -75,13 +75,10 @@ struct string_traits<wchar_t>{
 class StringUtils {
  public:
   /**
-   * Converts a string to a boolean
-   * Better handles mixed case.
+   * Checks and converts a string to a boolean
    * @param input input string
-   * @param output output string.
+   * @returns an optional of a boolean: true if the string is "true" (ignoring case), false if it is "false" (ignoring case), nullopt for any other value
    */
-  static bool StringToBool(std::string input, bool &output);
```

Comment:
This is good, but `StringToBool()` is still there in the cpp file, and it is also called from one place.  Type `git grep StringToBool` to find all usages.

Reply:
Yeah I am not sure why I left that part. Anyway, fixed it now :)